src/scripts/main.js

```diff
@@ -1,7 +1,26 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const box = document.createElement('div');
+  const head = document.createElement('h2');
```

Comment:
```suggestion
  const notification = document.createElement('div');
  const notificationTitle = document.createElement('h2');
```
 try to give them more descriptive names